client/Gruntfile.js

```diff
@@ -16,7 +16,26 @@ module.exports = function (grunt) {
         tasksDir: 'tasks',
         bowerDir: 'bower',
         poDir: 'po',
-        livereloadPort: 35729
+        livereloadPort: 35729,
+        ngtemplates: {
```

Comment:
you can put this into `tasks/options/ngtemplates.js` it will pick it from there.